protocols/kad/src/kbucket.rs

```diff
@@ -650,7 +650,7 @@ mod tests {
     fn rand_distance() {
         fn prop(ix: u8) -> bool {
             let d = BucketIndex(ix as usize).rand_distance(&mut rand::thread_rng());
-            let n = U256::from(<[u8; 32]>::from(d.0));
+            let n = d.0;
```

Comment:
```suggestion
            let n = U256::from_bug_endian(d.0.to_big_endian());
```

I assume the redundant conversion was there for testing purposes?

Reply:
d.0 is of type `U256`, this seems to be only testing `U256` from an extern crate

Reply:
As I understand, the roundtrip conversion exists because the old API uses big-endian implicitly, which has been fixed in the new release

Reply:
okay